registry/infrastructure/models.py

```diff
@@ -151,18 +151,3 @@ class ServiceReviewHistory(Base):
     reviewed_on = Column("reviewed_on", TIMESTAMP(timezone=False))
     created_on = Column("created_on", TIMESTAMP(timezone=False))
     updated_on = Column("updated_on", TIMESTAMP(timezone=False))
-
```

Comment:
This is needed. I will clean this up, once monitoring service is wired in the ease with publisher portal design